### ISSUE TITLE
Accordion test failures

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#671]
After changes introduces by a156e55, accordion tests started to fail. That's because Accordion header's `aria-controls` attribute is set only one the accordion panel is visible. If not, it's removed indeed. 

However Accordion tests still make a deep use of the aforementioned attribute, bringing tests to fails.

**Comment by wkeese** [comment]
Thanks, I pushed your fixes as fb0c1eb5c89bb165662a1c8a6c50faa8f0f03083 and 0b614c689e613e32839708cd0570ac2ee09a9103.
